data/src/main/java/me/androidbox/data/local/converter/AttendeeConverter.kt

```diff
@@ -0,0 +1,22 @@
+package me.androidbox.data.local.converter
+
+import androidx.room.TypeConverter
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.Types
+import me.androidbox.data.local.entity.AttendeeEntity
+
+class AttendeeConverter {
+    private val moshi = Moshi.Builder().build()
+    private val attendeeListType = Types.newParameterizedType(List::class.java, AttendeeEntity::class.java)
+    private val jsonAdapter = moshi.adapter<List<AttendeeEntity>>(attendeeListType)
+
+    @TypeConverter
+    fun fromJson(json: String): List<AttendeeEntity> {
+        return jsonAdapter.fromJson(json) ?: emptyList()
+    }
+
+    @TypeConverter
+    fun toJson(listOfAttendee: List<AttendeeEntity>): String {
+        return jsonAdapter.toJson(listOfAttendee)
+    }
+}
```

Comment:
If you save attendees and photos as JSON, you don't need to make them separate tables of your database, they can simply be plain classes